PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -2366,20 +2395,17 @@ struct HfTrackIndexSkimCreatorCascades {
   PROCESS_SWITCH(HfTrackIndexSkimCreatorCascades, processNoCascades, "Do not skim HF -> V0 cascades", true);
 
   void processCascades(SelectedCollisions const& collisions,
-                       aod::BCsWithTimestamps const&,
-                       // soa::Filtered<aod::V0Datas> const& V0s,
-                       aod::V0Datas const& V0s,
+                       aod::V0Datas const& v0s, // TODO: I am now assuming that the V0s are already filtered with my cuts (David's work to come)
                        FilteredTrackAssocSel const& trackIndices,
-                       TracksWithPVRefitAndDCA const& tracks
+                       TracksWithDCA const& tracks,
 #ifdef MY_DEBUG
-                       ,
-                       aod::McParticles& mcParticles
+                       aod::McParticles& mcParticles,
 #endif
-                       ) // TODO: I am now assuming that the V0s are already filtered with my cuts (David's work to come)
+                       aod::BCsWithTimestamps const&)
   {
     // set the magnetic field from CCDB
     for (const auto& collision : collisions) {
-      auto bc = collision.bc_as<o2::aod::BCsWithTimestamps>();
+      auto bc = collision.template bc_as<o2::aod::BCsWithTimestamps>();
```

Comment:
Why is `.template` needed here? What is the template argument involved?

Reply:
Indeed there is no need (reverted)